src/robot/conf/languages.py

```diff
@@ -307,6 +309,7 @@ class En(Language):
     variables_setting = 'Variables'
     documentation_setting = 'Documentation'
     metadata_setting = 'Metadata'
+    name_setting = "Name"
```

Comment:
It's interesting that the setting has name `Name` here but it still needs to be `Suite Name` in data. Most likely this is due to lexer having `Suite Name`. To make sure the used name matches, existing translation tests should also use `Name` setting. It ought to be enough to use `Nimi` (the Finnish translation) in `atest/testdata/parsing/translations/finnish/tests.robot` and enhance `Validate Translations` keyword in `atest/robot/parsing/translations.robot` so that it validates `${SUITE.name}` similarly as e.g. `${SUITE.doc}`.

Reply:
Changed to `Name` in  data